src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -68,6 +68,7 @@ import Disclaimer from '../../components/Disclaimer';
 import SecondaryColumn from './SecondaryColumn';
 import styles from './ArticlePage.styles';
 import { ComponentToRenderProps, TimeStampProps } from './types';
+import ReadMoreButton from './ReadMoreButton';
```

Comment:
Not sure if we should call this "read more" how about 'show more' as that is what we are doing??